package.json

```diff
@@ -9,12 +9,11 @@
   "dependencies": {
     "@types/jest": "^23.3.10",
     "jest-zone-patch": "^0.0.8",
-    "ts-jest": ">=23.10 <23.11"
+    "ts-jest": "~23.10.0"
   },
   "devDependencies": {
-    "@types/node": "^10.12.10",
```

Comment:
I think yarn.lock should get updated as well?

Reply:
Yeah it should :smile: